lab_1/main.py

```diff
@@ -12,9 +24,23 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    tokens = []
+    if type(text) == str:
+        text = text.lower()
+        for symbol in text:
```

Comment:
Do you really need to go for each symbol in text if you replace only specific symbols?